### PR TITLE
CM-84 IVDCMModule settings

### PR DESCRIPTION
**Pull request by VladSemenyuk**

### CODE REVIEW

**Comment by VladSemenyuk** [comment]
@bonewell

File: src/appMain/smartDeviceLink.ini [hunk 1]
```diff
@@ -214,3 +214,8 @@ InteriorVDCapabilitiesFile = ./plugins/InteriorVehicleDataCapabilities.json
 address = 127.0.0.1
 port = 8092
 timeout_period_seconds = 10
+
+[IVDCM]
```
Comment (DKlimenko) [inline src/appMain/smartDeviceLink.ini]: please add description

File: src/appMain/smartDeviceLink.ini [hunk 2]
```diff
@@ -214,3 +214,8 @@ InteriorVDCapabilitiesFile = ./plugins/InteriorVehicleDataCapabilities.json
...
+ip = 127.0.0.1
+port = 5050
```
Comment (DKlimenko) [inline src/appMain/smartDeviceLink.ini]: Is it possible to provide getters for these params in profile?

**Comment by bonewell** [comment]
Reviewed.

File: src/components/config_profile/include/config_profile/profile.h [hunk 3]
```diff
@@ -498,6 +498,10 @@ class Profile : public utils::Singleton<Profile> {
 
     bool logs_enabled() const;
 
+    std::string ivdcm_ip() const;
```
Comment (DKlimenko) [inline src/components/config_profile/include/config_profile/profile.h]: reference

File: src/components/config_profile/include/config_profile/profile.h [hunk 4]
```diff
@@ -498,6 +498,10 @@ class Profile : public utils::Singleton<Profile> {
...
+    uint16_t    ivdcm_port() const;
+    std::string ivdcm_nic_name() const;
```
Comment (DKlimenko) [inline src/components/config_profile/include/config_profile/profile.h]: refe

**Comment by DKlimenko** [comment]
reviewed

**Pull request merged**